ext/SymbolicsLuxCoreExt.jl

```diff
@@ -0,0 +1,11 @@
+module SymbolicsLuxCoreExt
+
+using LuxCore, Symbolics
+
+@register_array_symbolic LuxCore.stateless_apply(
+    model::LuxCore.AbstractExplicitLayer, x::AbstractArray, ps::Union{NamedTuple, <:AbstractVector}) begin
+    size = LuxCore.outputsize(model)
```

Comment:
This needs to change based on the new LuxCore PR

Reply:
I'm not sure how to get the `rng` here 🤔

Reply:
This already works for layers that can define the 1 arg `outputsize`, like `Chain`, but it would be nice to be able to use the general form.

Reply:
LuxCore.outputsize(model, x, Xoshiro(0))

Reply:
We don't have `Random` in deps, so we would need to either add as a passive dep or make the extension available only if `Random` is loaded (which would be quite confusing). I opted for the passive dep, but I'm not sure if it's okay to load the package only in the extension.

Reply:
I don't mind making the default in LuxCore as `outputsize(layer, x, rng = _default_rng())`

Reply:
No need, I forgot that I can also use stuff from LuxCore in here  😅